docs/reference/func_build.md

```diff
@@ -57,7 +57,7 @@ func build
 
 ```
       --build-timestamp        Use the actual time as the created time for the docker image. This is only useful for buildpacks builder.
-  -b, --builder string         Builder to use when creating the function's container. Currently supported builders are "pack" and "s2i". ($FUNC_BUILDER) (default "pack")
+  -b, --builder string         Builder to use when creating the function's container. Currently supported builders are "pack", "s2i" and "host". ($FUNC_BUILDER) (default "pack")
```

Comment:
Should we list here if it is not in the active code path?

Reply:
True; probably not yet.  That change must have snuck in from a later request.